### ISSUE TITLE
Remove unnecessary code

### ISSUE THREAD

**Issue by AngelUMICH** [AngelUMICH/Long-Time-Dynamics#83]
### Details
The potential table files and their associated variables and arrays and functions are no longer needed since we use LAMMPS to calculate energies and forces using their methods. The potential files can be removed along with the pos_init.dat and pos_ref.dat files. This issue can only started after issues #25 and #82 are completed.

### Definition of Done
- [x] Removed not needed variables and arrays
- [x] Removed not needed related functions
- [x] Removed the use for FINAL.f

**Comment by AngelUMICH** [comment]
In the dimer class the dimer force function is not being used.
Subroutine RSP in OPLD subroutine will no longer be needed as well (after #85 is completed).

**Comment by AngelUMICH** [comment]
the final.f file is no longer needed /  is not being used either.
